ui/i18n/src/main/res/values/strings.xml

```diff
@@ -381,7 +381,7 @@
     <string name="download_pref_details">Details</string>
     <string name="import_export_pref">Import/Export</string>
     <string name="import_export_search_keywords">backup, restore</string>
-    <string name="appearance">Appearance</string>
+    <string name="appearance">Theming</string>
```

Comment:
Please use a new string name to avoid the mismatch between title and content. Only change the English strings file.

Reply:
@ByteHamster sorry for the stupid question, but before misunderstandings arise.
Can I simply change it to:
`<string name="theming">Theming</string>` and change the UI to `<PreferenceCategory android:title="@string/theming">` or should I leave the existing one as it is and create a new one?

Reply:
Yes you can simply change it :) (I think there is no other place that uses the same string)

Reply:
Thank you for the hint. Reworked!